.github/workflows/docker.yml

```
@@ -23,7 +26,15 @@ jobs:
             --build-arg BUILDKIT_INLINE_CACHE=1 \
             --cache-from "dependabot/dependabot-core:latest" \
             .
-      - name: Push image to packages
+      - name: Push image to packages (latest)
+        if: '!contains(github.ref, "refs/tags")'
         run: |
           docker login -u ${{ secrets.DOCKER_USERNAME }} -p ${{ secrets.DOCKER_PASSWORD }}
           docker push "dependabot/dependabot-core:latest"
+      - name: Push image to packages (tagged)
+        if: contains(github.ref, "refs/tags")
+        run: |
+          docker login -u ${{ secrets.DOCKER_USERNAME }} -p ${{ secrets.DOCKER_PASSWORD }}
+          VERSION="$(grep -Eo "[0-9]+\.[0-9]+\.[0-9]+" common/lib/dependabot/version.rb)"
+          docker tag "dependabot/dependabot-core:latest" "dependabot/dependabot-core:$VERSION"
+          docker push "dependabot/dependabot-core:$VERSION"
```

Added `docker logout` here https://github.com/dependabot/dependabot-core/pull/2430/files for fork PRs :neckbeard: 

```suggestion
          docker push "dependabot/dependabot-core:$VERSION"
          docker logout
```